Adding Universal Sentence Encoder Model

Akhil-Chalam opened the pull request:
Hello Naveen,
I am updating the Universal Sentence Encoder model's implementation to the project.,

Akhil-Chalam committed:
Adding Universal Sentence Encoder Model

naveentds reviewed (approved):
@Akhil-Chalam Great work on adding the Universal encoder. Code looks good to me.
I have one comment. Can you update the Results section for Universal Encoder in README.md file. Then, you can go ahead and merge the PR.

Akhil-Chalam committed:
Update README.md

Hello Naveen,
I have updated the README file to show the results of applying the Universal Sentence Encoder model.
